RasterPropMonitor/Core/MonitorPage.cs

```diff
@@ -576,11 +577,12 @@ public void RenderBackground(RenderTexture screen)
                     }
                     break;
                 case BackgroundType.Texture:
+                    //call clear before redraw of textures
                     GL.Clear(true, true, ourMonitor.emptyColorValue);
                     Graphics.DrawTexture(new Rect(0, 0, screen.width, screen.height), backgroundTexture);
                     break;
                 case BackgroundType.Handler:
-
+                    //No clear here as it would interfere with the handlers(Causing effects such as VesselViewer to blink)
```

Comment:
There needs to be an "else" block added to the inner if clause with a GL.Clear in it to handle the case where backgroundHandlerMethod returns false, but there's no signal texture, or noSignalTexture is false.  To see why, look at a ship that has no docking port and no target, and select the docking port cam view on the MFD.  In pseudo-code:

```
if(!backgroundHandlerMethod)
{
    if(texture && showNoSignal)
    {
        Graphics.DrawTexture
    }
    else
    {
        GL.Clear
    }
}
```